packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -228,7 +231,10 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
         userId={localParticipant.userId}
         renderElement={
           localVideoStream?.renderElement ? (
-            <StreamMedia videoStreamElement={localVideoStream.renderElement} />
+            <>
+              {renderLocalCameraSwitcher && <LocalVideoCameraCycleButton />}
```

Comment:
Why are we not passing any props to the `LocalVideoCameraCycleButton` component?

Reply:
Coming to a PR near you soon..